src/lottie-player.ts

```diff
@@ -385,6 +397,11 @@ export class LottiePlayer extends LitElement {
   }
 
   private _render(): void {
+    const dpr = this.renderConfig?.devicePixelRatio || window.devicePixelRatio || 1;
```

Comment:
`or` operand is correct? What will be dpr value, if  

```const dpr  = 8 || 4 ```

Please check this, we can consider applying the DPR option clearly.

```
dpr = 1;

//boolean
if (this.renderConfig?.devicePixelRatio) {   
   dpr = window.devicePixelRatio;
}
```

Reply:
In JS, that's not a bitwise operator. `or` operand with non-boolean value checks if the value is null or undefined.

see also: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR

```js
// 1. check renderConfig.devicePixelRatio, if null or zero, step 2, returns value otherwise.
// 2. check window.devicePixelRatio, if browser doesn't support DPR, step 3, returns value otherwise
// 3. returns `1`, meaning default value of this scope is `1`
this.renderConfig?.devicePixelRatio || window.devicePixelRatio || 1;
```

So the `const dpr  = 8 || 4` returns 8.

`||` this is actually called "logical or", For the "bitwise or" we can go with `|` (single pipe).

Reply:
@tinyjin Got it, however I am talking about the bitwise issue, the priority matter here. 4 || 8? Is it right the bigger value must be overridden with? 

One another considerable problem is, user have no idea the best value of the `devicePixelRatio`. they should not designate the value but leave it to the system environment  => `window.devicePixelRatio`.

Reply:
Change the option to `enableDevicePixelRatio(boolean)` to leave control to the browser system.